app/views/casework/send_decisions/new.html.erb

```diff
@@ -26,16 +28,27 @@
     <% if @decisions.all? &:complete? %>
       <%= form_with model: @form_object, url: crime_application_send_decisions_path do |f| %>
 
-        <%= f.govuk_radio_buttons_fieldset :next_step, legend: { tag: 'h2', size: 'm' } do %>
-
-          <%= f.govuk_radio_button :next_step, :send_to_provider, link_errors: true %>
-          <%= f.govuk_radio_button :next_step, :add_another %>
+        <% if @crime_application.work_stream == 'non_means_tested' %>
```

Comment:
I'm wondering if its time for a "TypeOfApplication" concern that include as we're defining this in a few places:

https://github.com/ministryofjustice/laa-review-criminal-legal-aid/blob/6e8daffb77cd4305f5db4a2f9399bba24563d06a/app/aggregates/reviewing/available_reviewer_actions.rb#L65-L75

https://github.com/ministryofjustice/laa-review-criminal-legal-aid/blob/6e8daffb77cd4305f5db4a2f9399bba24563d06a/app/models/crime_application.rb#L193-L199